packages/init/src/steps/starters/slices.ts

```diff
@@ -37,25 +35,35 @@ export async function sendSlicesFromStarter(
   );
   spinner.start();
 
-  const acl = await createAcl(endpoints.AclProvider, repository, authorization);
+  const acl: Acl = await client.createAcl().catch((error: ClientError) => {
+    writeError(
+      "Uploading screenshots for your slices failed, please contact us."
+    );
+    writeError(error.message, "Full error:");
+    process.exit(1);
```

Comment:
Should it fail hard when screenshots are not uploaded?

Reply:
That's a good point, i'mm remove the exit process.
I thought that if creating the ACL fails, it's most likely something we'd want to know about immediately to fix it.

You're right tho, it's part of the screenshots.